galore/cli/galore_get_cs.py

```diff
@@ -54,28 +54,33 @@ def get_parser():
         Space-separated symbols for elements in material.""")
 
     parser.add_argument('--dataset', type=str,
-                        help='You can enter "Scofield" or "Yeh"')
+                        help=
+        """You can enter 'Scofield' or 'Yeh' """)
```

Comment:
This isn't clearer than the previous formatting, I think autopep8 is sneaking in again?

Reply:
Would still be good to use `choices` here, then Argparse is responsible for handling bad user input.